### ISSUE TITLE
Change the names of the default toggles

### ISSUE THREAD

**Issue by rappc87** [merdok/homebridge-miot#579]
Can you add a section to change the names of the default toggles? That way we don't have to reorganize them after every reboot.

**Comment by merdok** [comment]
This is something I plan to do in the future yes, just not sure when I will have the time to tackle that...

**Comment by rappc87** [comment]
> This is something I plan to do in the future yes, just not sure when I will have the time to tackle that...

Thank you very much. this is a really important feature because it is very tiring to edit the names over and over again every time.

So is there a method that I can do it manually for now?

**Comment by merdok** [comment]
No, currently there is no method to do it manually. A proper way needs to be provided to save the set name to the settings and restore that after a reboot, but that requires a lot of changes...

**Comment by rappc87** [comment]
> No, currently there is no method to do it manually. A proper way needs to be provided to save the set name to the settings and restore that after a reboot, but that requires a lot of changes...

so I guess the best way is to wait. I hope you can find a time and add it as soon as possible. thank you very much for your work. Hava a good one
